subworkflows/local/utils_nfcore_taxprofiler_pipeline/main.nf

```diff
@@ -203,11 +202,17 @@ def toolCitationText() {
         params.shortread_qc_tool == "fastp" ? "fastp (Chen et al. 2018)." : "",
     ].join(' ').trim()
 
+
+    def text_shortread_redundancy = [
+        "Short-read reference-free metagenome coverage estimation was performed with Nonpareil (Rodriguez-R et al. 2018)."
+    ].join(' ').trim()
+
     def text_longread_qc = [
         "Long read preprocessing was performed with:",
-        !params.longread_qc_skipadaptertrim ? "Porechop (Wick et al. 2017)," : "",
-        !params.longread_qc_skipqualityfilter ? "Filtlong (Wick 2021)," : "",
-        "."
+        params.longread_adapterremoval_tool == "porechop_abi" ? "Porechop_ABI (Bonenfant et al. 2023)," : "",
+        params.longread_adapterremoval_tool == "porechop" ? "Porechop (Wick et al. 2017)," : "",
+        params.longread_filter_tool == "filtlong" ? "Filtlong (Wick 2021)," : "",
```

Comment:
Should this string be `Filtlong (Wick 2021).`, i.e. replace the comma with a period? Since you can only choose Filtlong or Nanoq?

Reply:
Ooooh nice catch!

Reply:
https://github.com/nf-core/taxprofiler/pull/530/commits/e6a44cf1898450def08bd923aaa24ed6f1420309